packages/fluentui/react-builder/src/config.tsx

```diff
@@ -538,6 +548,85 @@ export const renderJSONTreeToJSXElement = (
   });
 };
 
+const packageImportList: Record<string, CodeSandboxImport> = {
+  '@fluentui/react-icons-northstar': {
+    version: projectPackageJson.version,
+    module: FUIIcons,
+    required: false,
+  },
+  '@fluentui/react-northstar': {
+    version: projectPackageJson.version,
+    module: FUI,
+    required: true,
+  },
+};
+
+export const JSONTreeToImports = (tree: JSONTreeElement, imports = {}) => {
+  if (tree.props?.icon) {
+    const iconModule =
+      (tree.moduleName === '@fluentui/react-northstar' && '@fluentui/react-icons-northstar') || 'ErrorNoPackage';
+    if (imports.hasOwnProperty(iconModule)) {
+      if (!imports[iconModule].includes(tree.props?.icon.type)) {
+        imports[iconModule] += `${tree.props?.icon.type}, `;
+      }
+    } else {
+      imports[iconModule] = `${tree.props?.icon.type}, `;
+    }
+  }
+  if (tree.moduleName && tree.$$typeof === 'Symbol(react.element)') {
+    if (imports.hasOwnProperty(tree.moduleName)) {
+      if (!imports[tree.moduleName].includes(tree.displayName)) {
+        imports[tree.moduleName] += `${tree.displayName}, `;
+      }
+    } else {
+      imports[tree.moduleName] = `${tree.displayName}, `;
+    }
+  }
+
+  tree.props?.children?.forEach(item => {
+    if (typeof item !== 'string') {
+      imports = JSONTreeToImports(item, imports);
+    }
+  });
+  return imports;
+};
+
+export const getCodeSandboxInfo = (tree: JSONTreeElement, code: string) => {
+  const imports = JSONTreeToImports(tree);
+  let codeSandboxExport = 'import * as React from "react";\n';
+  const packageImports: Record<string, CodeSandboxImport> = {
+    '@fluentui/code-sandbox': {
+      version: sandboxPackageJson.version,
+      module: CodeSandbox,
+      required: true,
+    },
+    react: {
+      version: projectPackageJson.peerDependencies['react'],
+      module: React,
+      required: true,
+    },
+    'react-dom': {
+      version: projectPackageJson.peerDependencies['react-dom'],
+      module: ReactDOM,
+      required: true,
+    },
+    prettier: {
+      version: docsComponentsPackageJson.peerDependencies['prettier'],
+      module: null,
+      required: true,
+    },
+  };
+  for (const [module, components] of Object.entries(imports)) {
+    codeSandboxExport += `import {${components}} from "${module}";\n`;
+    packageImports[module] = packageImportList[module];
+  }
+  codeSandboxExport += '\n export default function example() { \n return (\n';
```

Comment:
```suggestion
  codeSandboxExport += '\n export default function Example() { \n return (\n';
```

BTW, we can use there template strings: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals